README.md

```diff
@@ -220,3 +229,33 @@ This may be better suited for you if you plan to change some of the automaticall
 ## Recap
 
 After following this complete README, we should have an ETH smart contract as well as a Starknet smart contract, both connected to act as a bridge between these two chains.
+
+## Upgrade Contracts in Testnet
+
+### Ethereum
+After deploying the `YABTransfer` contract, you can perform upgrades using the `make ethereum-upgrade` command.
```

Comment:
```suggestion
After deploying the `YABTransfer` contract, you can perform upgrades using the following command:
`make ethereum-upgrade`
```
I think commands are more clear and more easily identifiable if they are on their own line